### DISCUSSION TITLE
[Solved] [X96 Air P3] No HDMI output, no SSH

### DISCUSSION THREAD

**Discussion by ramirez22-fr** [devmfc/debian-on-amlogic#56] [Q&A] [answered]
Hello.

First congrats for the work, it's amazing !
Second, sorry for my poor English, I hope I will be understood.

I flashed a SDcard with the latest release.
I uncommented the correct line in boot.config
I put the SDcard in the box and follow instructions to boot. Linux boot perfectly on SDcard, but I have no HDMI output.
I can connect through SSH (PuTTY) with the IP indicated on the LCD screen of the box. All seem run perfectly.

I try to install system on eMMC and launch a script (don't remember the name, but it had explicit tittle). But an error occurs, indicate that it's necessary to launch anothe script before for "multiboot" (hope my memory is good).
I launch it and reboot the device.

Now, impossible to reconnect by SSH (and always no HDMI ...).
The system seem runing cause LCD screen indicate correct time (at the start, time is not correct but maybe a NTP server is called, cause time is correct few second after), temperature and IP.
I try to reset SSH key but without improuvement.

I have this laconic message from PuTTY : "Network error : Connection refused"

I Try to remove SDcard and restart the device, but nothing run, I stay on the X96 start picture.

Is it possible to help me ?

**Comment by devmfc** [comment, marked as answer]
X96 Air P3 is my "reference" box. So all my development tests are done on that box. So the chance that HDMI of the box does not work is very small.

-Try another screen or try without attaching anything on HDMI
-try a better power supply
-try a different sd card or better still, a USB flash disk.

But you say that the vfd display is showing the time, temperature and IP, this tells me not only that the kernel did boot, but also that user space is running. Maybe the ip address changed? Can you see that on your router or scan the local network? 

Did you by any chance installed some other image that replaced the vendor bootloader? If so, try to restore stock Android.

If none of this works, you will need to connect a serial adapter to the board to see what is going on.

**Comment by ramirez22-fr** [comment]
Hello,

Many thanks for your answer. 

I have already flash a stock ROM with succes, the box is running like before.
For the screen, I have a UWQHD, maybe output box not compliant.

I will try your suggestions and repport result here.
Thanks again



[EDIT]
Ok, after few test, I find that the TVbox was not compliant with my KVM and/or with UWQHD screen ...
So I used a new HDMI cable, a lower res. screen, a new SDcard (and SDCard reader) and finaly, all is perfect.

So:
Flash the SDcard with debian distro
Modify boot.config
First launch with reset buton
Launch script mal-multiboot-setup.sh
Reboot
Launch script aml-install-to-emmc.sh
Halt
Reemove SDCard
Power on the box, linux running like a charme !

Many thanks for help and for your job !!!
